Sources/WhisperKit/Core/Utils.swift

```diff
@@ -135,6 +144,10 @@ func tokenizerNameForVariant(_ variant: ModelVariant) -> String {
     return tokenizerName
 }
 
+func isModelMultilingual(logitsDim: Int?) -> Bool {
```

Comment:
We have this already here: https://github.com/argmaxinc/WhisperKit/blob/cf75348b3bd0a51cfdffbfaa7553df6a5b4e6bdf/Sources/WhisperKit/Core/Models.swift#L60

Thoughts on combining them? I think checking the logitDims is more robust, perhaps it can be set on the model or the textdecoder on load here: https://github.com/argmaxinc/WhisperKit/blob/cf75348b3bd0a51cfdffbfaa7553df6a5b4e6bdf/Sources/WhisperKit/Core/WhisperKit.swift#L313